docs/heasarc/heasarc.rst

```diff
@@ -268,7 +268,23 @@ for ``'box'`` and:
     >>> Heasarc.query_region(catalog='xmmmaster', spatial='polygon',
                   polygon=[(226.2,10.6),(225.9,10.5),(225.8,10.2),(226.2,10.3)])
 
-for ``'polygon'``.
+for ``'polygon'``. For ``'all-sky'``:
+
+.. doctest-skip::
+
+    >>> Heasarc.query_region(pos, spatial='all-sky', catalog='csc', maxrec=None)
+
+though you may find that maxrec has a hard limit of 1e5 regardless of how you set it. See `issue #3387 tracker <https://github.com/astropy/astroquery/issues/3387>`_. 
+
```

Comment:
no need to add xref to astroquery issue. once that one is fixed the docs can be updated.